### ISSUE TITLE
Option --warning-regex is ignored

### ISSUE THREAD

**Issue by matlion** [wummel/linkchecker#604]
The option --warning-regex or -W is ignored when using LinkChecker.
I'm using LinkChecker 9.3 with Python 2.7.9 (both manually installed).

In this example call "LinkChecker" should be reported:

./linkchecker -r 1 -WLinkChecker http://wummel.github.io/linkchecker/
INFO 2015-06-20 09:47:28,047 MainThread Checking intern URLs only; use --check-extern to check extern URLs.
LinkChecker 9.3              Copyright (C) 2000-2014 Bastian Kleineidam
LinkChecker comes with ABSOLUTELY NO WARRANTY!
This is free software, and you are welcome to redistribute it
under certain conditions. Look at the file `LICENSE' within this
distribution.
Get the newest version at http://wummel.github.io/linkchecker/
Write comments and bugs to https://github.com/wummel/linkchecker/issues
Support this project at http://wummel.github.io/linkchecker/donations.html

Start checking at 2015-06-20 09:47:28+002
 1 thread active,     0 links queued,    0 links in   0 URLs checked, runtime 1 seconds
 9 threads active,     0 links queued,   24 links in   8 URLs checked, runtime 6 seconds

Statistics:
Downloaded: 5KB.
Content types: 7 image, 10 text, 0 video, 0 audio, 1 application, 0 mail and 15 other.
URL lengths: min=22, max=76, avg=49.

That's it. 33 links in 17 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2015-06-20 09:47:37+002 (9 seconds)

**Comment by matlion** [comment]
I just downgraded to 8.2
Here warning-regex works as expected.

**Comment by code-tree** [comment]
I can confirm this bug too.

However, I did get regex check to work via the linkcheckerrc with:
```
[RegexCheck]
warningregex=searchterm
```

**Comment by dpalic** [comment]
Thank you for the issue report. Sadly this project is dead, and a new team is around with https://github.com/linkcheck/linkchecker 
for more details please see: #708
**Also please close this issue** and report it freshly on the new repo https://github.com/linkcheck/linkchecker/issues if your issue still persists

**Comment by matlion** [comment]
Won't fix
